code/modules/mob/living/simple_animal/friendly/fox.dm

```diff
@@ -22,12 +22,66 @@
 	var/turns_since_scan = 0
 	var/mob/living/movement_target
 
+/mob/living/simple_animal/pet/fox/New()
+	if(!vore_organs.len)
+		var/datum/belly/B = new /datum/belly(src)
```

Comment:
tbqh this is done so as not to modify Polaris code and preserve easy mergability.